### DISCUSSION TITLE
User/IP authentication not working

### DISCUSSION THREAD

**Discussion by nicedevil007** [NginxProxyManager/nginx-proxy-manager#2223] [Q&A]
Hey guys,

I have my NPM setup for internal webpage that already have a basic authentication.
Now I wanted to try out the basic auth from NPM and a restriction to allowed IPs.

When I just enable the allowed IPs it is working. If I enable the user authentication with a username and password it asks me to enter my credentials all the time when `Satisfy any` is enabled.

If I enter the credentials I setup in NPM I get the window again for authentication and I guess I have then to enter the credentials from the webservices behind... not working.

So I tryed to debug it without any luck. The `access.log` just shows me 401 errors all day long (so I guess the authentication isn't working at all with username / password from NPM)? I also took a look at the created proxy-host file with correct entries (satisfy any, IP allow thing and basic auth thing were added). I also opened the access list file with the username and the hashed password.... seems all correct.

Anything I'm missing to solve the issue?

**Comment by zensea** [comment]
I have encountered the same issue and look forward to getting the correct answer.

**Comment by zensea** [comment]
I didn't realize this was a question from two years ago, yet I'm still encountering it today.
